docs/local_development_with_docker.md

```diff
@@ -86,6 +87,30 @@ Use `invoke docker-npm update`.
 You don't need to rebuild the `watch-static-files` image.
 
 
+---
+
+## Connecting Docker to your code editor
+
+
+### Pycharm
+
+This feature is only available for the professional version of Pycharm. Follow the official instructions [available here](https://www.jetbrains.com/help/pycharm/using-docker-as-a-remote-interpreter.html#config-docker)
+
+### Visual Studio Code
+
+Visual Studio Code use a feature called Dev Container to run Docker projects. The configuration files are in the `.devconatainer` directory. This feature is only available starting VSCode 1.35 stable. For now, we're only creating a python container to get Intellisense, we're not running the full project inside VSCode.
```

Comment:
Can you add some info about why "For now" (like what we're blocked on)?

Reply:
Chris tried to set up docker compose for vscode and made some progress but the workflow was sadly terrible :( The "for now" is me acknowledging that the setup could be better and being okay to revisit it if Docker support for vscode gets better.